src/vm.rs

```diff
@@ -534,6 +579,57 @@ impl Vm {
         }
     }
 
+    /// Asynchronously runs an exported wasm function in a (named or active) [module instance](crate::Instance).
```

Comment:
Refine the comments: `Asynchronously runs an exported wasm function in a (named or active) [module instance](crate::Instance) with a timeout setting.`

Thanks!